### ISSUE TITLE
Measure current duplicate submission rates 1wk post launch of New Confirmation Page [12.18]

### ISSUE THREAD

**Issue by pacerwow** [department-of-veterans-affairs/va.gov-team#98359]
### Issue Description
As a Product Manager
I want to analyze post-launch records of duplicate disability claim submissions
So that I can evaluate if the new confirmation page reduces duplicate submissions compared to the baseline

Dependent on 
- #97845

### Tasks

- [ ] 

### Acceptance Criteria:
- Query database for duplicate submissions 1 week post-launch (same or similar strategy used for (#97845)
- Compare against pre-launch baseline metrics
- Document findings in this ticket and share in DSVA Slack
- Highlight any statistically significant changes (What are we considering to be sta
- Identify if duplicates decreased for submissions using new confirmation page

### Additional Information
One nuance to this is that we want to be able to be confident the change didn't happen by chance, so while this reads as 1wk, we should consider the timing of 5103 & Claim Certification, as we want to take this measurement _before_ then, so we're able to rule out other factors that might influence the results. 

PM would like to run this data back, measuring the P-value - @pacerwow can do once the data is collected.

**Comment by Dottifacilities** [comment]
Blocked by https://github.com/department-of-veterans-affairs/va.gov-team/issues/97845

**Comment by sethdarragile6** [comment]
Using [the script](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/benefits/scripts/526/super-duper-duped-user-submissions-reporter.rb) developed for #97845 we compared the % of duplicate submission from a week leading up to the New Confirmation Page release (Dec. 11) with the week after:

- 12/04 through 12/10: 1.19% (188 out of 15754 submissions)
- 12/11 through 12/17: 0.34% (48 out of 13935 submissions)

This would seem to be an almost 70% drop in duplicate submissions! This appears to be dramatically less than the monthly and quarterly baseline metrics detailed [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/engineering_research/duplicate_submission_metrics.md) as well, which range from 1.23-1.38%.

Looking at that period [in DataDog](https://vagov.ddog-gov.com/apm/resource/vets-api-sidekiq/sidekiq.job/31f0dbe2e343475e?query=%40_top_level%3A1%20env%3Aeks-prod%20service%3Avets-api-sidekiq%20operation_name%3Asidekiq.job%20resource_name%3A%22EVSS%3A%3ADisabilityCompensationForm%3A%3ASubmitForm526AllClaim%22&compareVersionEnd=0&compareVersionPaused=false&compareVersionStart=0&env=eks-prod&fromUser=false&spanType=service-entry&summary=qson%3A%28data%3A%28visible%3A%21t%2Cchanges%3A%28%29%2Cerrors%3A%28selected%3Acount%29%2Chits%3A%28selected%3Acount%29%2Clatency%3A%28selected%3Alatency%2Cslot%3A%28agg%3A95%29%2Cdistribution%3A%28isLogScale%3A%21f%29%2CshowTraceOutliers%3A%21t%29%2Csublayer%3A%28slot%3A%28layers%3AserviceAndInferred%29%2Cselected%3Apercentage%29%2ClagMetrics%3A%28selectedMetric%3A%21s%2CselectedGroupBy%3A%21s%29%29%2Cversion%3A%211%29&topGraphs=latency%3Alatency%2Chits%3Acount%2Cerrors%3Arate%2CbreakdownAs%3Apercentage&traces=qson%3A%28data%3A%28%29%2Cversion%3A%210%29&start=1733299200000&end=1734508799999&paused=true) for SubmitForm526AllClaim, there was a pretty big spike in errors over the weekend for the pre-deploy interval compared to the post-deploy interval. This, in combination with just generally reduced traffic over the post-deploy interval, could explain the seeming disparity.

However, I looked at slightly larger 2wk intervals before and after the Dec. 11th launch, and found essentially the same basic ratio, even as the sample size for the post-release interval increased:

- 11/27 through 12/10: 1.25% (324 out of 25978 submissions)
- 12/11 through 12/25: 0.43% (115 out of 26877 submissions)

So, in conclusion... did duplicates decrease for submissions using the new confirmation page? The answer _seems_ to be "Yes". But before taking _too_ much stock in these results, I'd recommend more future and past query runs to compare against. December can be a bit of an atypical month for everybody and in many ways.
